cypress/integration/userCanLogin.feature.js

```diff
@@ -55,7 +55,21 @@ describe("User can log in, when clicking the 'my reQuest' link", () => {
       cy.get("#email").type("user@mail.com");
       cy.get("#password").type("wrongpassword");
       cy.get("#submit-btn").contains("Submit").click();
-      cy.get("#error-message").should("contain","Invalid login credentials. Please try again.");
+      cy.get("#error-message").should(
+        "contain",
+        "Invalid login credentials. Please try again."
+      );
+    });
+  });
+  it("successfully logs out", () => {
+    cy.get("#login-form").within(() => {
+      cy.get("#email").type("user@mail.com");
+      cy.get("#password").type("password");
+      cy.get("#submit-btn").contains("Submit").click();
+    });
+    cy.get("#welcome-and-logout").within(() => {
+      cy.get("button#logout").click();
+      cy.get("button#logout").should("not.exist");
```

Comment:
And `p` should not contain 'user@mail.com'

Reply:
done